.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/sigs.k8s.io/controller-runtime"]
```

Comment:
as chatted offline, probably don't need this now.

Reply:
Yep! Removed.